.github/ISSUE_TEMPLATE/change_proposal.yaml

```
@@ -1,5 +1,5 @@
-name: Propose changes to existing conventions
-description: Propose changes you'd like to be added to existing conventions
+name: Propose enhancement to existing conventions
+description: Propose changes you'd like to contribute
```

otherwise could be interpreted that they should only open issue if they are planning to send PR to implement
```suggestion
description: Propose changes you'd like to see
```